edx_proctoring/templates/emails/proctoring_attempt_satisfactory_email.html

```diff
@@ -9,8 +9,7 @@
     {% blocktrans %}
         Your proctored exam "{{ exam_name }}" in
         <a href="{{ course_url }}">{{ course_name }}</a> was reviewed and you
-        met all exam requirements. You can view your grade on the course
-        progress page.
+        met all exam requirements.
```

Comment:
I created https://openedx.atlassian.net/browse/EDUCATOR-404 for conditionally adding back in progress page text/links.